### PR TITLE
GRW-382 / FAQ accordion jumpy

### PR DESCRIPTION
**Pull request by amandaflod**
## What?

Change the margin on the accordion item to padding, as the margin together with the margin that comes from the p element in the description were making things jumpy (thanks @robinandeer! 👑)

## Why?

**Ticket(s): [GRW-382]**

https://user-images.githubusercontent.com/59652803/173345854-e9fa71a2-3277-4e0d-b9d1-238e18a151b0.mov

[GRW-382]: https://hedvig.atlassian.net/browse/GRW-382?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by simonauner — APPROVED** [review]
Nice, I tried to fix that several times but didn't manage to get it working for some reason! ⭐

File: src/client/pages/OfferNew/FaqSection.tsx [hunk 1]
```diff
@@ -43,8 +43,7 @@ const TabsWrapper = styled.div`
 
 const StyledAccordionItem = styled(AccordionItem)`
   &:not(:last-child) {
-    margin-bottom: 1.5rem;
-
+    padding-bottom: 1.5rem;
     ${LARGE_SCREEN_MEDIA_QUERY} {
       margin-bottom: 2.5rem;
```
Comment (simonauner) [inline src/client/pages/OfferNew/FaqSection.tsx:48]: Shouldn't this also be padding instead of margin?

Comment (amandaflod) [reply src/client/pages/OfferNew/FaqSection.tsx:48]: Yes, will add that! Thank you :)) Although for some reason it looked fine on bigger screens with the margin 🤔 But nice with consistency!

**Review by simonauner — APPROVED** [review]

**Comment by amandaflod** [comment]
We really have Robin to thank for this one 😊 I was driving myself crazy clicking on that accordion haha

**Review by guilhermespopolin — APPROVED** [review]
Was the problem those crazy rules for margin collapse? 🤔 
I always remember [this article](https://www.joshwcomeau.com/css/rules-of-margin-collapse/) when something crazy happens with margins.

**Comment by simonauner** [comment]
> Was the problem those crazy rules for margin collapse? 🤔 I always remember [this article](https://www.joshwcomeau.com/css/rules-of-margin-collapse/) when something crazy happens with margins.

It's a known problem with react-animate-height: https://github.com/Stanko/react-animate-height#collapsing-margins

**Comment by guilhermespopolin** [comment]
> > Was the problem those crazy rules for margin collapse? 🤔 I always remember [this article](https://www.joshwcomeau.com/css/rules-of-margin-collapse/) when something crazy happens with margins.
> 
> It's a known problem with react-animate-height: https://github.com/Stanko/react-animate-height#collapsing-margins

Oh I see. Good to know about it. Thanks 🙌🏻

**Pull request merged**